Force lftp to use just ncurses (rather than both Solaris curses and ncurses)
because of the conflicting definition of setupterm:

/usr/include/curses.h
  extern int setupterm(char *, int, int *);

/usr/include/ncurses/term.h
  extern NCURSES_EXPORT(int) setupterm (NCURSES_CONST char *,int,int *);

https://github.com/lavv17/lftp/pull/760

--- lftp-4.9.3/src/lftp_tinfo.cc
+++ lftp-4.9.3/src/lftp_tinfo.cc
@@ -20,7 +20,7 @@
 #include <config.h>
 
 extern "C" {
-#if defined(HAVE_CURSES_H)
+#if defined(HAVE_CURSES_H) && !defined(__sun__)
 # include <curses.h>
 # if defined(HAVE_TERM_H)
 #  include <term.h>
